spectator-api/src/main/java/com/netflix/spectator/api/ManualClock.java

```diff
@@ -42,16 +43,23 @@ public ManualClock() {
   public ManualClock(long wallInit, long monotonicInit) {
     wall = new AtomicLong(wallInit);
     monotonic = new AtomicLong(monotonicInit);
+    countPolled = new AtomicLong(0);
   }
 
   @Override public long wallTime() {
+    countPolled.incrementAndGet();
     return wall.get();
   }
 
   @Override public long monotonicTime() {
+    countPolled.incrementAndGet();
     return monotonic.get();
   }
 
+  public long countPolled() {
```

Comment:
I would have to think about this as it would be a new public user-facing API. I'm not sure how often this will come up as we haven't used it before. Also, if it does come up again it might be necessary to separate out which timing call was made.

How about just creating a custom sub-class that tracks this and is part of the test cases for now. We can revisit in the future if there is enough need.